dockerfiles/xenial/Dockerfile

```diff
@@ -22,31 +20,19 @@ RUN apt-get update && apt-get -y install \
     libyaml-dev \
     libzookeeper-mt-dev \
     protobuf-compiler \
-    # dh-virtualenv only works with python2 - https://github.com/spotify/dh-virtualenv/issues/122
-    python \
-    python3.6-dev \
+    python3.7-dev \
+    python3.9-dev \
```

Comment:
why do we need 3.9 too?

Reply:
dependency of latest dh_virtualenv which is a deb build for ubuntu 22

Reply:
dropped it as it works with 3.7 too and i'm force-installing the deb anyway